src/commands/fav.rs

```diff
@@ -487,7 +487,7 @@ pub async fn block(ctx: &Context, msg: &Message, args: Args) -> CommandResult {
 #[only_in("guilds")]
 #[description = "Creates a list of all favs on the server"]
 #[allowed_roles("Mods")]
-pub async fn create_fav_list(ctx: &Context, msg: &Message, args: Args) -> CommandResult {
+pub async fn create_fav_list(ctx: &Context, msg: &Message) -> CommandResult {
```

Comment:
Ich bin mir ziemlich sicher dass es hier knallt weil die Funktionssignatur nicht passt, wenn ein Parameter unbenutzt ist wird man die warnung los in dem man den Parameter mit `_` ersetzt oder beginnen lässt (z.b. `_args`)

Reply:
Ich glaube das ist kein Problem. Habe etwas getestet, und das klappt. [Hier in einem Beispiel von serenity](https://github.com/serenity-rs/serenity/blob/fb1f0bfdcefada4a08863d5e389df44737a86b12/examples/e05_command_framework/src/main.rs#L293) haben die auch ein `command` mit nur zwei Argumenten.